images/base/scripts/common/setup.sh

```diff
@@ -83,7 +83,12 @@ fi
 
 # First install protoc
 cd /tmp
-git clone https://github.com/google/protobuf.git
+if [ x$MACHINE = xs390x ]
+then
+    git clone https://github.com/gongsu832/protobuf.git
```

Comment:
IIRC, you can clone a specific tag/branch... why not do that rather than clone and then checkout?